mscore/plugin/api/score.cpp

```diff
@@ -100,5 +109,14 @@ Measure* Score::lastMeasure()
       {
       return wrap<Measure>(score()->lastMeasure(), Ownership::SCORE);
       }
+//---------------------------------------------------------
+//   Score::firstMeasure
```

Comment:
Just a small thing, but the comment doesn't match the function's name ;-)

Reply:
outch, bad copy/paste error

Reply:
fixed ;-)